core-features/generate-client-install-on-sub-project-npm/run.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -eux
+
+npm install
+
+cp -r ../_common/generate-client-install-on-sub-project/* .
+
+cp package.json sub-project
+
+cd sub-project
+
+npm install
+npm remove @prisma/client
+
+env DEBUG="prisma:generator" npx prisma generate
```

Comment:
comments why we do these things would be nice

Reply:
Done